changelog/v0.31.0/ginkgo-v2.yaml

```diff
@@ -0,0 +1,55 @@
+changelog:
+  - type: NON_USER_FACING
+    issueLink: https://github.com/solo-io/gloo/issues/7222
+    resolvesIssue: false
+    description: >
+      Upgrade Ginkgo from v1 to v2, using https://onsi.github.io/ginkgo/MIGRATING_TO_V2 as a helpful guide.
+  - type: BREAKING_CHANGE
+    issueLink: https://github.com/solo-io/gloo/issues/7222
+    resolvesIssue: false
+    description: >
+      Remove multicluster package, as it was not used by any projects
+  - type: DEPENDENCY_BUMP
```

Comment:
how did you choose which dependency bumps to call out, here?

Reply:
I probably could have added more. I tried to callout internal libraries, or important libraries that affected many files. Are there any deps that you think I should call out?